src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/MIReplicaReportingServiceTest.java

```diff
@@ -891,6 +960,27 @@ private TaskResource createAndSaveThisTask(String taskId, String taskName,
         return taskResourceRepository.save(taskResource);
     }
 
+    private TaskResource createAndSaveThisTask(String taskId, String taskName,
+                                               CFTTaskState taskState, String lastAction, String assignee) {
+        TaskResource taskResource = new TaskResource(
+            taskId,
+            taskName,
+            "someTaskType",
+            taskState,
+            "987654",
+            OffsetDateTime.parse("2023-04-05T20:15:45.345875+01:00")
+        );
+        taskResource.setCreated(OffsetDateTime.parse("2023-03-23T20:15:45.345875+01:00"));
+        taskResource.setPriorityDate(OffsetDateTime.parse("2023-03-26T20:15:45.345875+01:00"));
+        taskResource.setLastUpdatedAction(lastAction);
+        taskResource.setLastUpdatedTimestamp(OffsetDateTime.parse("2023-03-29T20:15:45.345875+01:00"));
+        taskResource.setJurisdiction("someJurisdiction");
+        taskResource.setLocation("someLocation");
+        taskResource.setRoleCategory("someRoleCategory");
+        taskResource.setAssignee(assignee);
+        return taskResourceRepository.save(taskResource);
+    }
```

Comment:
As a general rule it's better to put test object creators into their own class so it can be re-used by many tests.  
Private methods like this just mean that someone else will probably write something very similar if not the same in another class.